test/web-platform-tests/index.js

```diff
@@ -228,48 +236,84 @@ const runWebPlatformTest = require("./run-web-platform-test")(exports, path.reso
   "XMLHttpRequest/open-url-base.htm",
   "XMLHttpRequest/open-url-base-inserted.htm",
   "XMLHttpRequest/open-url-base-inserted-after-open.htm",
+  // "XMLHttpRequest/open-url-bogus.htm", // todo
+  // "XMLHttpRequest/open-url-encoding.htm", // todo
   "XMLHttpRequest/open-url-fragment.htm",
   "XMLHttpRequest/open-url-javascript-window-2.htm",
   "XMLHttpRequest/open-url-javascript-window.htm",
+  "XMLHttpRequest/open-url-multi-window.htm",
+  "XMLHttpRequest/open-url-multi-window-2.htm",
+  "XMLHttpRequest/open-url-multi-window-3.htm",
+  // "XMLHttpRequest/open-url-multi-window-4.htm", // todo
```

Comment:
In general if you could leave more informative comments on why a test fails in jsdom that would be nice. I see you do that in a few places. If it's too much trouble to debug I understand.